internal/service/alarms/serve.go

```diff
@@ -91,6 +89,9 @@ func Serve(config *api.AlarmsServerConfig) error {
 		if err != nil {
 			return fmt.Errorf("failed to parse global cloud id: %w", err)
 		}
+	} else {
+		slog.Debug("No globalCloudID set by the operator, generating one instead")
```

Comment:
We shouldn't be generating our own here.   The user must set this for the system to work as designed.   We should leave this as the nil UUID if not set... and continue to block the creation of new subscription until the user sets it.

Reply:
Yeah i added this just to quickly test things end to end. removing this.